shell/platform/windows/compositor_opengl.cc

```diff
@@ -0,0 +1,143 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/compositor_opengl.h"
+
+#include "GLES3/gl3.h"
+#include "flutter/shell/platform/windows/flutter_windows_view.h"
+
+namespace flutter {
+
+namespace {
+
+// The metadata for an OpenGL framebuffer backing store.
+struct FramebufferBackingStore {
+  uint32_t framebuffer_id;
+  uint32_t texture_id;
+};
+
+}  // namespace
+
+CompositorOpenGL::CompositorOpenGL(FlutterWindowsEngine* engine,
+                                   impeller::ProcTableGLES::Resolver resolver)
+    : engine_(engine), resolver_(resolver) {}
+
```

Comment:
The `CompositorOpenGL` mirrors Linux's backing store provider: https://github.com/flutter/engine/blob/101396fd3b8216acea95f09fa7c3fc8e97b38ab8/shell/platform/linux/fl_backing_store_provider.cc#L19-L102